packages/frontend/src/components/DashboardTiles/TileForms/ChartUpdateModal.tsx

```diff
@@ -0,0 +1,125 @@
+import { DashboardTile } from '@lightdash/common';
+import {
+    Button,
+    Flex,
+    Group,
+    Modal,
+    ModalProps,
+    Select,
+    Stack,
+    Title,
+} from '@mantine/core';
+import { useForm } from '@mantine/form';
+import { IconChartAreaLine } from '@tabler/icons-react';
+import produce from 'immer';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useChartSummaries } from '../../../hooks/useChartSummaries';
+import { useDashboardContext } from '../../../providers/DashboardProvider';
+import MantineIcon from '../../common/MantineIcon';
+
+interface ChartUpdateModalProps<T> extends ModalProps {
+    tile: T;
+    onClose: () => void;
+    onConfirm?: (tile: T) => void;
+}
+
+const ChartUpdateModal = <T extends DashboardTile>({
+    tile,
+    onClose,
+    onConfirm,
+    ...modalProps
+}: ChartUpdateModalProps<T>) => {
+    const form = useForm({
+        initialValues: {
+            uuid: tile.properties.savedChartUuid,
```

Comment:
TS error here:
`TS2339: Property 'savedChartUuid' does not exist on type '{ title: string | null; hideTitle?: boolean | undefined; savedChartUuid: string | null; } | { title: string; hideTitle?: boolean | undefined; url: string; } | { title: string; content: string; }'.   Property 'savedChartUuid' does not exist on type '{ title: string; hideTitle?: boolean | undefined; url: string; }'.`